ads/adhese.md

```diff
@@ -0,0 +1,66 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Adhese
+
+Adhese adserver.
+More information about us can be found on our [website](https://www.adhese.com).
+
+## Example
+
+### Basic setup
+
+```html
+<amp-ad width=300 height=250
+    type="adhese"
+    data-location="_sdk_amp_"
+    data-position=""
+    data-format="amprectangle"
+    data-account="demo"
+    data-requestType="ad">
+</amp-ad>
+```
+
+### With additional parameters
+
+```html
+<amp-ad width=300 height=250
+    type="adhese"
+    data-location="_sdk_amp_"
+    data-position=""
+    data-format="amprectangle"
+    data-account="demo"
+    data-requestType="ad"
+    json='{"targeting":{"br": ["sport", "info"],"dt": ["desktop"]}}'>
+</amp-ad>
+```
+
+
+## Configuration
+
+See ad network documentation or contact Adhese support.
+
+Required parameters:
+
+- `data-account`
+- `data-request_type`
```

Comment:
Correct. I'm testing on ads.adhese.amp.max.html where I use data-request-type.
Forgot to make the same change to adhese.md.